<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainUI</class>
 <widget class="QMainWindow" name="MainUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>294</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Archive:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_archive">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeWidget" name="tree_contents">
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_progress_icon">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_progress">
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>403</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionAdd_File"/>
    <addaction name="actionAdd_Dirs"/>
    <addaction name="actionRemove_File"/>
    <addaction name="separator"/>
    <addaction name="actionExtract_All"/>
    <addaction name="actionExtract_Sel"/>
   </widget>
   <widget class="QMenu" name="menuBurn_toDisk">
    <property name="title">
     <string>&amp;Burn to Disk</string>
    </property>
    <addaction name="actionUSB_Image"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuBurn_toDisk"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="actionAdd_File"/>
   <addaction name="actionAdd_Dirs"/>
   <addaction name="actionRemove_File"/>
   <addaction name="separator"/>
   <addaction name="actionExtract_All"/>
   <addaction name="actionExtract_Sel"/>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open Archive</string>
   </property>
   <property name="statusTip">
    <string>Open archive</string>
   </property>
  </action>
  <action name="action_New">
   <property name="text">
    <string>&amp;New Archive</string>
   </property>
   <property name="statusTip">
    <string>New archive</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionAdd_File">
   <property name="text">
    <string>Add File(s)</string>
   </property>
   <property name="statusTip">
    <string>Add files to archive</string>
   </property>
  </action>
  <action name="actionRemove_File">
   <property name="text">
    <string>Remove File(s)</string>
   </property>
   <property name="statusTip">
    <string>Remove selection from archive</string>
   </property>
  </action>
  <action name="actionExtract_All">
   <property name="text">
    <string>Extract All</string>
   </property>
   <property name="statusTip">
    <string>Extract archive into a directory</string>
   </property>
  </action>
  <action name="actionAdd_Dirs">
   <property name="text">
    <string>Add Directory</string>
   </property>
   <property name="statusTip">
    <string>Add directory to archive</string>
   </property>
  </action>
  <action name="actionExtract_Sel">
   <property name="text">
    <string>Extract Selection</string>
   </property>
   <property name="statusTip">
    <string>Extract Selected Items</string>
   </property>
  </action>
  <action name="actionUSB_Image">
   <property name="text">
    <string>USB Image</string>
   </property>
   <property name="statusTip">
    <string>Copy an IMG to a USB device (may require admin permission)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
